<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveImageForm</class>
 <widget class="QWidget" name="MoveImageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBoxYDir">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-999999</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y方向の移動量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X方向の移動量</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMaster">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>マスター画像</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右・下方向がプラス</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTarget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>検査画像</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxXDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-999999</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="DisplayImage.qrc"/>
 </resources>
 <connections/>
</ui>
